packages/engines/README.md

```diff
@@ -0,0 +1,10 @@
+# `@prisma/engines`
+
+This package ships the Prisma Engines, namely the Query Engine, Migration Engine, Introspection Engine and Prisma Format.
+It has a one to one mapping to the engine releases in its version.
+In the postinall hook of this package, all engines available for the current platform are downloaded from the Prisma CDN.
```

Comment:
```suggestion
This package is used to download the Prisma Engines, namely the Query Engine, Migration Engine, Introspection Engine and Prisma Format.
Its version usd to have has a one to one mapping to the engine hash in its version.
Since Prisma 4 its version follows the `prisma/prisma` monorepo versioning and it uses the `@prisma/engines-version` package to map to the engines hash.

In the `postinstall` hook of this package, all engines available for the current platform are downloaded from the Prisma CDN `https://binaries.prisma.sh`.
```